typing/ctype.ml

```diff
@@ -1947,15 +1947,18 @@ let rec estimate_type_layout env ty =
    it's a valid argument to [t].  (We believe there are still loops like this
    that can occur, though, and may need a more principled solution later).
 *)
-let rec constrain_type_layout ~fixed env ty1 layout2 fuel =
+let rec constrain_type_layout ~reason ~fixed env ty1 layout2 fuel =
+  (* XXX ASZ: Are all the [~reason]s in this function the same? *)
```

Comment:
Our convention is that we don't merge `XXX` comments. Maybe just change this to `CR-soon` or something?

Reply:
My impression was our convention is not to merge `XXX`s to `main`, but it feels reasonable to merge them to the unboxed types branch.

Reply:
(And indeed if we want to merge this but also make sure this issue is resolved before merging to main, leaving a `Layouts XXX` feels the best thing to do?)

Reply:
Yes, I agree. My impression was that this particular comment was OK to merge to `main`, which is why I suggested the change. That is, I was worried about a second merge of this code, not the merge to `unboxed-types-v1`.